### PR TITLE
Update `sass-embedded` info in `README.md`

### PR DESCRIPTION
**Pull request by ntkme**
`sass-embedded` just had [first stable release 1.0.0](https://rubygems.org/gems/sass-embedded/versions/1.0.0). Removing the note about it being experimental.

### CODE REVIEW

**Comment by ashmaroli** [comment]
How about adding a recommendation to use `sass-embedded-1.x` instead of older versions?

**Comment by ntkme** [comment]
@ashmaroli Updated. It is a bit wordy now. Please feel free to make amendments.

**Review by ashmaroli — APPROVED** [review]

**Comment by ashmaroli** [comment]
Thanks @ntkme 
@jekyllbot: merge +doc
